<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>1200</width>
      <height>750</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>295</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Центр</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinCenterX">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="prefix">
     <string>X:</string>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinCenterY">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="prefix">
     <string>Y:</string>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupColor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>-10</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="radioFront">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Фон</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioGreen">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Зеленый</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBlue">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Синий</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioWhite">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>95</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Белый</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioYellow">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Желтый</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioRed">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Красный</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioAqua">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Аквамарин</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupOkr">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="radioBiblio">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Библиотечная функция</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioSrDot">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Алг. средней точки</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioParam">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Параметрическое ур-е</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioKanon">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Каноническое ур-е</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBrez">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Брезенхейм</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupRAB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="radioR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Окружность</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioAB">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Эллипс</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="spinBoxR">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="prefix">
     <string>R:</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxA">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>450</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="prefix">
     <string>a:</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxB">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="prefix">
     <string>b:</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDrawOkr">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нарисовать окружность</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDrawSpec">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>690</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нарисовать спектор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>570</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxN">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="prefix">
     <string>N:</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Спектр, количество колец</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
